app/src/models/banner.ts

```diff
@@ -92,6 +94,18 @@ export type Banner =
       /** number of commits squashed */
       readonly commitsCount: number
     }
+  | {
+      readonly type: BannerType.SuccessfulReorder
+      /** number of commits reordered */
+      readonly count: number
+      /** callback to run when user clicks undo link in banner */
+      readonly onUndo: () => void
+    }
+  | {
+      readonly type: BannerType.ReorderUndone
+      /** number of commits reordered */
+      readonly commitsCount: number
+    }
```

Comment:
Maybe we could have just one banner for all the multi commit operations too? 🤔

Reply:
Yeah. I think I was heading that way.. but didn't get there on the refactor. :P  I am all for it.